backend/src/meetings/meeting/meeting.controller.ts

```diff
@@ -146,23 +150,63 @@ export class MeetingController {
     return this.meetingsService.getBookingsByMeetingID(meetingID).then(attendees => attendees.map(BookingEntity.toDTO));
   }
 
-  @Get(":mid/attendee/:username/attend")
-  async getAttendesUsername(@Param('mid') mid : string, @Param('username') username : string){
-
+  @Put(":mid/attendee/:username/attend")
+  async getAttendesUsername(@Param('mid') mid : string, @Param('username') username : string): Promise<BookingDto>{
```

Comment:
getAttendesUsername is not a good name. attendToMeeting or bookMeeting would be better